Tests/IGListStackSectionControllerTests.m

```diff
@@ -85,8 +85,9 @@ - (void)test_whenInitializingStack_thatNumberOfItemsMatches {
     section4.items = 1;
 
     IGListStackedSectionController *stack = [[IGListStackedSectionController alloc] initWithSectionControllers:@[section1, section2, section3, section4]];
-
-    XCTAssertEqual([stack numberOfItems], 6);
+    [stack performBatchAnimated:false updates:^{} completion:^(BOOL finished) {
+        XCTAssertEqual([stack numberOfItems], 6);
```

Comment:
Yep, it should have dawned on me I made the changed tests asynchronous :).  The issue with these particular tests is that they don't follow the normal pattern of the other tests because they don't use the adapter.  I don't know without digging further why they don't use the adapter or if they can be changed easily.  Also, my proposed solution wouldn't work anyway since the completion block never gets called because there's no CollectionContext set on the section controller.  One easy way to use to fix the broken tests would be to use the following pattern:

```
- (void)test_whenInitializingStack_thatNumberOfItemsMatches {
    IGListTestSection *section1 = [[IGListTestSection alloc] init];
    section1.items = 2;
    IGListTestSection *section2 = [[IGListTestSection alloc] init];
    section2.items = 3;
    IGListTestSection *section3 = [[IGListTestSection alloc] init];
    section3.items = 0;
    IGListTestSection *section4 = [[IGListTestSection alloc] init];
    section4.items = 1;

    IGListStackedSectionController *stack = [[IGListStackedSectionController alloc] initWithSectionControllers:@[section1, section2, section3, section4]];
    [stack performSelectorOnMainThread:@selector(reloadData) withObject:nil waitUntilDone:TRUE];
    XCTAssertEqual([stack numberOfItems], 6);
}
```

Basically just call the private reloadData method using performSelector.  Let me know what you think the best way to proceed would be.